src/ray/core_worker/task_manager.cc

```diff
@@ -1026,7 +1021,8 @@ bool TaskManager::RetryTaskIfPossible(const TaskID &task_id,
                                  spec.AttemptNumber(),
                                  RayConfig::instance().task_oom_retry_delay_base_ms())
                            : RayConfig::instance().task_retry_delay_ms();
-    retry_task_callback_(spec, /*object_recovery*/ false, update_seqno, delay_ms);
+    retry_task_callback_(
```

Comment:
After this change, the sequence numbers of actor tasks become strictly increasing.

Reply:
A possible follow-up: refactor `retry_task_callback_`.